### PR TITLE
fix: fix Node implemented types wrongly adding an id argument everywhere they are defined

### PR DESCRIPTION
**Pull request by bellini666**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/guides/relay.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ Or to retrieve the first 10 fruits available:
 
 ```graphql
 query {
-  fruitConnection(first: 10) {
+  fruits(first: 10) {
```
Comment (patrick91) [inline docs/guides/relay.md:162]: 👍

**Review by patrick91 — APPROVED** [review]
I think this is much better thanks!

**Comment by botberry** [comment]
# Pre-release
:wave:

Releasing commit [5d112adb9f7537036d65fd8fe26d1496c37a14d6] to PyPi as pre-release! :package:

**Comment by botberry** [comment]
# Pre-release
:wave:

Pre-release **0.183.1.dev.1686081894** [5d112adb9f7537036d65fd8fe26d1496c37a14d6] has been released on PyPi! :rocket:
You can try it by doing:

```shell
poetry add strawberry-graphql==0.183.1.dev.1686081894
```

File: docs/guides/relay.md (hunk 1 above)
Comment (bellini666) [reply docs/guides/relay.md:162]: We forgot about this =P

**Pull request merged**